CMakeLists.txt

```diff
@@ -11,6 +11,14 @@ set(CMAKE_CXX_EXTENSIONS OFF)
 find_package(PythonExtensions REQUIRED)
 find_package(Cython REQUIRED)
 
+include(FetchContent)
+FetchContent_Declare(openroad
+  GIT_REPOSITORY    https://github.com/The-OpenROAD-Project/OpenROAD.git
+  GIT_TAG           81706aef55de40a6f2eccadf25c20f418a554ab6
```

Comment:
Looks good. Maybe you left this for later, but a hard coded hash in the cmake file makes me really uncomfortable. Can we collect these magic constants in one place? 

We have an attempt with the version number here:
https://github.com/siliconcompiler/siliconcompiler/blob/main/siliconcompiler/_metadata.py

Reply:
Yes we can (and should), I have a "mockup" of one way to handle it. We can either merge this in or wait (since there is no urgency on this) to find a "limited" magic number solution

Reply:
In retrospect, once @WRansohoff verifies that the changes to CMakeLists doesn't break the install, I think we will merge this. The change to remove the "magic" numbers will come in a separate PR since there may be some nice additional things we can do with that which would be orthogonal to this PR.

Reply:
That build is currently running, and I just kicked off [a 'wheels' action](https://github.com/siliconcompiler/siliconcompiler/actions/runs/3913966431) which will attempt to build the package on all common platforms.

Reply:
Looks like the Wheels action may need updating to remove the submodule reference; I made a `remove-tools-submodules-wheels` branch to test that.

The local `pip install -e .` process works on a clean Ubuntu machine after updating `pyopenssl` and installing `bison`, but it only seems to install the Python package, not the command-line entry points such as `sc` and `sc-show`.

Reply:
Per our offline discussion, it looks like the command-line entry points do get installed, but in a different location: `~/.local/bin/`